merge.py

```diff
@@ -46,7 +50,12 @@ def get_wikidata_frame():
 	    }
 	"""
 
+	# TODO In case https://github.com/RDFLib/sparqlwrapper/pull/225 is merged
```

Comment:
just cross-linking https://github.com/RDFLib/sparqlwrapper/pull/225 for reference here